js/samples/quickstart-kotlin-azfunction/app/src/main/assets/immersiveReader.html

```diff
@@ -0,0 +1,41 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License. -->
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <script type="text/javascript" src="https://contentstorage.onenote.office.net/onenoteltir/immersivereadersdk/immersive-reader-sdk.1.1.0.js"></script>
+</head>
+<body>
+    <script type="text/javascript">
+        function handleLaunchImmersiveReader(message) {
+            if (!message) {
+                Android.showToast('Message is null or undefined!');
+            } else {
+                // Learn more about chunk usage and supported MIME types https://docs.microsoft.com/azure/applied-ai-services/immersive-reader/reference#chunk
+                var data = {
+                    title: message.content.title,
+                    chunks: message.content.chunks
+                };
+
+                // A simple declarative function used to close the Immersive Reader WebView via @JavaScriptInterface
+                var exitCallback = function() {
+                    Android.immersiveReaderExit();
+                }
+
+                // Learn more about options https://docs.microsoft.com/azure/applied-ai-services/immersive-reader/reference#options
+                var options = {
+                    cognitiveAppId: 'QuickStartKotlinSampleWebApp',
```

Comment:
cognitiveAppId - This is an Internal SDK-only param. Should be removed here

Reply:
removed